core/src/test/java/org/apache/calcite/sql/test/SqlOperatorBaseTest.java

```diff
@@ -5192,6 +5192,23 @@ private void checkNullOperand(SqlTester tester, String op) {
         "07000000789c4bad48cc2dc84905000bc002ed", "VARBINARY NOT NULL");
   }
 
+  @Test public void testUncompress() {
+    SqlTester sqlTester = tester(SqlLibrary.MYSQL);
+    sqlTester.checkNull("UNCOMPRESS(NULL)");
+    sqlTester.checkNull("UNCOMPRESS(x'')");
+
+    sqlTester.checkString("UNCOMPRESS(COMPRESS('test'))",
+        "test", "VARCHAR");
+
+    sqlTester.checkString("UNCOMPRESS(x'10000000789c4b4c44050033980611')",
+        "aaaaaaaaaaaaaaaa", "VARCHAR");
+
+    sqlTester.checkString("UNCOMPRESS(x'06000000789c2b4ecc2dc849050008de0283' )",
```

Comment:
I think in order to cover the test, we need to add a non-ByteString type test.

Reply:
The contract doestnot allow string type as an argument. Instead I have added another negative test case. Thanks :)